coops-ui/src/schemas/index.ts

```diff
@@ -0,0 +1 @@
+export * from './address-schema'
```

Comment:
Works for me, this was an inconsistency on my part. Will correct!

Reply:
Same thing here as above, since the mixins are classes they export cleaner, but i think we should export our constants, enums, interfaces and schemas etc as a batch. So when we use all of the above we don't have to refer to them as `someExportName.constant` or `someExportName.schema`. This is just my opinion on it, but we can discuss further!

Reply:
OK, I see only the mixins use a different format now. That's OK I guess.